src/xcode/ENA/ENA/Source/Services/CCLService/DCCWalletInfo.swift

```diff
@@ -0,0 +1,89 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import AnyCodable
+
+struct DCCWalletInfo: Codable, Equatable {
+
+	let admissionState: DCCAdmissionState
+	let vaccinationState: DCCVaccinationState
+	let boosterNotification: DCCBoosterNotification
+	let mostRelevantCertificate: DCCMostRelevantCertificate
+	let verification: DCCVerification
+	let validUntil: Date
+
+}
+
+struct DCCAdmissionState: Codable, Equatable {
+
+	let visible: Bool
+	let badgeText: DCCUIText?
+	let titleText: DCCUIText?
+	let subtitleText: DCCUIText?
+	let longText: DCCUIText?
+	let faqAnchor: String?
+
+}
+
+struct DCCVaccinationState: Codable, Equatable {
+
+	let visible: Bool
+	let titleText: DCCUIText?
+	let subtitleText: DCCUIText?
+	let longText: DCCUIText?
+	let faqAnchor: String?
+
+}
+
+struct DCCBoosterNotification: Codable, Equatable {
+
+	let visible: Bool
+	let identifier: String?
+	let titleText: DCCUIText?
+	let subtitleText: DCCUIText?
+	let longText: DCCUIText?
+	let faqAnchor: String?
+
+}
+
+struct DCCUIText: Codable, Equatable {
+
+	let type: String
+	let quantity: Double?
+	let quantityParameterIndex: Int?
+	let functionName: String?
+	let localizedText: [String: AnyCodable]?
+	let parameters: [String: AnyCodable]
+
+	func localized(languageCode: String? = Locale.current.languageCode) -> String? {
+		return localizedText?[languageCode ?? "de"]?.value as? String
```

Comment:
Is "de" correct here?
Does this sentence from the spec apply here? 
"If there is no property for the language code, en shall be used. If en is not available, de shall be used."

Reply:
This is just a dummy implementation that I needed to make the fake working. It will be replaced by Naveeds full implementation. So we'll have to check there if it does the .languageCode ?? "en" ?? "de" dance correctly 🕺😄